src/electionguard/election_polynomial.py

```diff
@@ -19,55 +22,68 @@
 )
 from .schnorr import make_schnorr_proof, SchnorrProof
 
+# TODO: rename to just polynomial and improve the documentation with some links to wikipedia
 
+# TODO: do not use lists here as they are susceptible to order-based attacks
 class ElectionPolynomial(NamedTuple):
     """
-    ElectionPolynomial is a polynomial defined by coefficients. The 0 coefficient is used for a secret key which can 
+    ElectionPolynomial is a polynomial defined by coefficients. 
```

Comment:
Idk if this is on purpose but a space here separates the description from the definition.

Also there are warnings for this but not using the name of the variable in the description.
`Polynomial defined by coefficients` is much simpler.